packages/libbpf/libbpf.0.1.0/opam

```diff
@@ -0,0 +1,52 @@
+opam-version: "2.0"
+synopsis: "Libbpf bindings"
+description: "Wrapped libbpf api's for writing BPF user programs in OCaml"
+maintainer: ["Lee Koon Wen"]
+authors: ["Lee Koon Wen"]
+license: ["ISC" "BSD-3-Clause"]
+tags: ["bindings" "bpf" "libbpf"]
+homepage: "https://github.com/koonwen/ocaml-libbpf"
+doc: "https://koonwen.github.io/ocaml-libbpf"
+bug-reports: "https://github.com/koonwen/ocaml-libbpf/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.13"}
+  "ctypes" {>= "0.22.0"}
+  "ppx_deriving"
+  "ppx_expect"
+  "conf-libbpf"
+  "conf-clang"
+  "conf-bpftool"
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/koonwen/ocaml-libbpf.git"
+# eBPF features by kernel version https://github.com/iovisor/bcc/blob/master/docs/kernel-versions.md
+# Fix to kernel >= 6.1 to provide bound BPF map types
+available: [ os = "linux" &
+	   (( os-distribution = "debian" & os-version >= "12" )    # Linux 6.1 & Libbpf 1.1.0
+ 	   |( os-distribution = "ubuntu" & os-version >= "23.04" ) # Linux 6.2 & Libbpf 1.1.0
+	   |( os-distribution = "fedora" & os-version >= "38" ))   # Linux 6.2 & Libbpf 1.1.0
+	   ]
```

Comment:
Eventually we'll probably want to have `conf-linux-kernel-version` packages so you can have more generic depends. But we shouldn't hold back this PR because of it.